test/mocks/courses.json

```diff
@@ -0,0 +1,35 @@
+[
+  {
+    "uuid": "d4e4e432-1c02-4d7f-817a-4fe6be1703c2",
+    "title": "Use the wireless PNG array, then you can input the auxiliary sensor!",
+    "description": "Try to index the THX alarm, maybe it will index the haptic transmitter!",
+    "permalink": "dolor-corrupti-non",
+    "date": 1504931975000,
+    "author": {
+      "name": "Nicolas Corwin",
+      "username": "Nicolas93"
+    }
+  },
+  {
+    "uuid": "d9c4adcf-07e0-492d-967b-80ed767f67c7",
+    "title": "If we synthesize the bandwidth, we can get to the AI driver through the wireless TCP driver!",
+    "description": "copying the driver won't do anything, we need to index the haptic XML transmitter!",
+    "permalink": "officia-qui-nemo",
+    "date": 1504795411000,
+    "author": {
+      "name": "Brandy Daniel",
+      "username": "Brandy64"
+    }
+  },
+  {
+    "uuid": "64db40ae-0ddb-49fe-94f9-36139da061cc",
+    "title": "If we reboot the port, we can get to the SAS alarm through the solid state EXE capacitor!",
+    "description": "You can't copy the matrix without quantifying the neural PCI interface!",
+    "permalink": "facere-similique-nam",
+    "date": 1504879901000,
+    "author": {
+      "name": "Katlyn Parisian",
+      "username": "Katlyn98"
+    }
+  }
+]
```

Comment:
du coup tu json ? et non markdown ?

Reply:
En fait, je pense que ça serait mieux qu'on puisse utiliser json pour lister les tutos (plus facile à parser que le MD), histoire d'avoir un accès facile aux metadata (`uuid`, `permalink`, `title`, `description`, `author`, ...). Et chaque tuto aura un dossier du nom du `permalink`, et dedans, il y aura des fichier md prédéfinis comme `home.md` et les `step1.md`, `step2.md`, `step3.md`, ...


```
├── tutos
|   ├── index.json
|   ├── fr-api-plateform
|   |   ├── home.md
|   |   ├── step1.md
|   |   ├── step2.md
|   |   ├── step3.md
|   |   └── step4.md
|   ├── en-symfony-angular
|   |   ├── home.md
|   |   ├── step1.md
|   |   ├── step2.md
|   |   └── step3.md
```

la liste des tutos (`tutos/index.json`) :

```json
{
  "tutos": [
    "fr-api-plateform",
    "fr-synfony-angular"
  ]
}
```

et l'index du tuto (`tutos/fr-api-plateform/index.json`):

```json
{
  "uuid": "d4e4e432-1c02-4d7f-817a-4fe6be1703c2",
  "title": "Use the wireless PNG array, then you can input the auxiliary sensor!",
  "description": "Try to index the THX alarm, maybe it will index the haptic transmitter!",
  "permalink": "fr-api-plateform",
  "date": 1504931975000,
  "author": {
    "name": "Nicolas Corwin",
    "username": "Nicolas93"
  }
}
```

Reply:
oups, je dis des bétises, je voulais dire que `tuto/index.json` contiendra la liste détaillée des tutos, comme dans les `mocks`, donc pas besoin des fichiers `index.json` dans chaque tuto